### DISCUSSION TITLE
Sprache in Issues, Code und Commits

### DISCUSSION THREAD

**Discussion by qwertfisch** [Rojetto/ReNeo#75] [General]
Bisher war der Code und seine Kommentare auf englisch, die Commit-Meldungen und die ganzen Issues auf deutsch. Für das Three-Layout sind nun zwei Issues auf englisch angelegt (soweit ok), aber die Arbeit daran übertrug sich auch auf die Commits.

Nun ist das bunt gemischt. Was ist mit zukünftigen Beiträgen und Issues? Alles nun auf englisch, oder ist deutsch weiterhin ok/erwünscht? Die allermeisten Commits werden wohl nicht das Three-Layout betreffen.

**Comment by Rojetto** [comment]
Berechtigter Punkt. Ich denke es macht Sinn, auf Nutzerseite beides zu erlauben/ermöglichen. Heißt konkret das Programm selber und Nutzerdokumentation (Readme) sollten auf Deutsch und Englisch verfügbar sein, und in Issues beides erlaubt.

Für die Entwicklung ist der Code ja schon auf Englisch. Da die Commit-Nachrichten relativ "nah" am Code sind und Nutzer mit diesen normalerweise wenig zu tun haben, ergibt es für mich Sinn, diese zukünftig auch auf Englisch zu schreiben. Falls tatsächlich jemand dem Projekt beitragen möchte und des Deutschen nicht mächtig ist (ich weiß, unwahrscheinlich), wirkt es damit weniger abschreckend.

Die Umstellungsphase für die Commit-Nachrichten muss man halt hinnehmen, aber es signalisiert zumindest, dass man dem gegenüber offen ist.

**Reply by qwertfisch** [reply]
Hm dann muss ich aufpassen, dass ich daran denke. In Projekten mit deutschsprachigen Mitarbeitern habe ich bisher nur deutsche Commitmeldungen verfasst (Code war natürlich auf englisch geschrieben und kommentiert). Im Endeffekt kam auch nie ein nicht-deutschsprachiger Kollege hinzu … Ich befürchte, das wird hier auch so sein.

Umstellungsphase? Die gibt es nicht. Man kann von heute auf morgen die Commits auch auf Französisch kommentieren. Das Problem ist die Historie, gerade wenn man nach bestimmten Dingen sucht. Wenn die Sprache nicht einheitlich ist, sucht man stets zweimal, zumindest geht es mir so. Wenn du nun (wieder) ausschließlich Englisch bevorzugst, werde ich mich bemühen, das so zu handhaben. Auf lange Sicht hin erlangt man dadurch auch wieder Einheitlichkeit.
